### PR TITLE
Use TEST_SUITE env var to drive specs

### PR DESCRIPTION
**Pull request by agrare**
This probably isn't the objectively best way to do this, however it does follow the TEST_SUITE pattern used by core and ui-classic and makes it easier for us to run with manageiq-cross_repo

https://github.com/ManageIQ/manageiq-cross_repo/pull/69

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
+language: ruby
+node_js: 12
+cache:
+  bundler: true
+  yarn: true
+addons:
+  postgresql: '10'
+rvm:
+- 2.5.8
+- 2.6.6
```
Comment (Fryguy) [inline .travis.yml:11]: Super minor, but I think we should keep the languages/versions together, so can this be moved up between the language and node_js key?

**Review by Fryguy — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (agrare) [reply .travis.yml]: :+1: done

**Comment by agrare** [comment]
Having a hell of a time getting the javascript tests to pass 😱

**Comment by agrare** [comment]
Okay got this running.  Instead of having `bundle exec rake` and the default rake task switching on the `ENV["TEST_SUITE"]` I added a `bin/ci/script.sh` so that the javascript tests don't have to `bin/setup` which takes an order of magnitude longer to `bundle install` and setup the database when it won't end up being used anyway.

**Pull request merged**